tests/api_connexion/endpoints/test_task_instance_endpoint.py

```diff
@@ -41,6 +41,11 @@
 DEFAULT_DATETIME_STR_2 = "2020-01-02T00:00:00+00:00"
 
 
+class _BadExecutorConfig:
+    def __str__(self):
+        raise Exception()
```

Comment:
```suggestion
class BadExecutorConfig:
    """This is used to test what happens when executor config serialization fails"""
    def __str__(self):
        raise Exception("Failed to render as string")
```

to give a little context how this obj is used

Reply:
If anything, we can create a custom exception class here. Somehting like `BadExecutorConfigException(AirflowException)`.